src/actions/EmailAction.js

```diff
@@ -119,7 +119,9 @@ module.exports = function(router) {
           label: 'Message',
           key: 'message',
           type: 'textarea',
-          defaultValue: '{{ table(form.components) }}',
+          defaultValue: '{{ table(form.components) }}' + '<br>' +
+          '<p><a href="{{ submissionLink }}">'
+          + 'View submission on form.io-view</a></p> ',
```

Comment:
People are not going to know what "form.io-view" is and we are now making this the default email that is sent out. I would say to just change this to "View submission on Form.io"